inductiva/fluids/__init__.py

```diff
@@ -9,17 +9,19 @@
     BEER,
     HONEY,
 )
-from .scenarios import DamBreak
-from .scenarios import FluidBlock
-from .scenarios import WindTunnel
-from .scenarios import (
+from .dam_break import DamBreak
+from .fluid_block import FluidBlock
+from .wind_tunnel import WindTunnel
+from .wind_terrain import WindOverTerrain
+from .fluid_tank import (
     FluidTank,
```

Comment:
This is a bit of a weird import.
Let's fix it later, in a subsequent PR.

Reply:
I agree... and I think it is more related with the general structure of imports that needs to be discussed.

Reply:
ok! Let's fix it later.